### DISCUSSION TITLE
Ubuntu 20 : Installation problem from the start

### DISCUSSION THREAD

**Discussion by mountaniol** [community/community#4028] [Codespaces]
Tried to install it on Ubuntu 20.01
Installation failed from the start:

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN checkPermissions Missing write access to /usr/local/lib
npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /usr/local/lib
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib'
npm ERR!  { [Error: EACCES: permission denied, access '/usr/local/lib']
npm ERR!   stack:
npm ERR!    'Error: EACCES: permission denied, access \'/usr/local/lib\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib' }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:

Doesn't look as a stable SW.

**Comment by 2percentsilk** [comment]
Hi there!

Could you provide some more context on what you were trying to do and what you expected to happen?

**Reply by mountaniol** [reply]
Hi there,

I just wanted to install it and study what and how.
Unfortunately, it ended with these errors.

Sincerely,
Sebastian.

**Reply by 2percentsilk** [reply]
Sorry I still don't quite understand - what were you trying to install? And were you trying to install it within a codespace?

**Comment by hwittenborn** [comment]
Any chance you didn't run the commands with `sudo`?
